### PR TITLE
 Testing PHPCS w/ re-enabled GitHub Workflow

### PR DESCRIPTION
**Pull request by borkweb**

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -944,7 +944,7 @@ public function init() {
 			$this->singular_event_label_lowercase             = tribe_get_event_label_singular_lowercase();
 			$this->plural_event_label_lowercase               = tribe_get_event_label_plural_lowercase();
 
-			$this->post_type_args['rewrite']['slug']            = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
+			$this->post_type_args['rewrite']['slug']          = $rewrite->prepare_slug( $this->rewriteSlugSingular, self::POSTTYPE, false );
```
Comment (tr1b0t) [inline src/Tribe/Main.php]: * :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Variable "post_type_args" is not in valid camel caps format
* :no_entry_sign: Equals sign not aligned correctly; expected 1 space but found 10 spaces
* :no_entry_sign: Object property "$rewriteSlugSingular" is not in valid snake_case format, try "$rewrite_slug_singular"
* :warning: Line exceeds 120 characters; contains 140 characters

<sub>hash: 11418f02ecd4d0382d0d3f9be7dc21bc</sub>

File: src/Tribe/Main.php [hunk 2]
```diff
@@ -961,6 +961,8 @@ public function init() {
 			$this->maybeSetTECVersion();
 
 			$this->run_scheduler();
+
+			$a = [ 1 ];
```
Comment (tr1b0t) [inline src/Tribe/Main.php]: :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: eb72efbc17be3ede5cf2c9302bbcf508</sub>

File: src/Tribe/Main.php (hunk 1 above)
Comment (bordoni) [reply src/Tribe/Main.php]: :fail:

Comment (tr1b0t) [inline src/Tribe/Main.php]: * :warning: Equals sign not aligned with surrounding assignments; expected 1 space but found 10 spaces
* :no_entry_sign: Object property "$rewriteSlugSingular" is not in valid snake_case format, try "$rewrite_slug_singular"

<sub>hash: 19e2434a0f39188f40b07e8e08191b42</sub>

**Comment by lucatume** [comment]
@bordoni comments make this look like a competition between him and the bot.
My money is on the bot.

**Pull request closed without merging**